projects/core/src/error-handling/cx-error-handler.ts

```diff
@@ -27,33 +26,26 @@ import {
  */
 @Injectable()
 export class CxErrorHandler implements ErrorHandler {
-  //TODO: Verify whether feature flag in this case is justified.
-  // if yes - Add Jira ticket to remove in next major release or after 6 months
-  private featureConfig = inject(FeatureConfigService);
-  logger = inject(LoggerService);
+  /**
+   * @deprecated Since 6.6
+   */
+  protected logger = inject(LoggerService);
+  protected injector = inject(Injector);
 
-  injector = inject(Injector);
-
-  errorInterceptors: ErrorInterceptor[] = sortErrorInterceptors(
+  protected errorInterceptors: ErrorInterceptor[] = sortErrorInterceptors(
     this.injector.get(ERROR_INTERCEPTORS, [])
   );
```

Comment:
[MEDIUM]
Why using `injector`? Can't we simply `inject(ERROR_INTERCEPTORS, [])`